### DISCUSSION TITLE
阿里云盘第三方客户端无法自动加载外置字幕

### DISCUSSION THREAD

**Discussion by 2607627233** [hooke007/mpv_PlayKit#473] [mpv 通用]
![image](https://github.com/hooke007/MPV_lazy/assets/43110396/40ebe084-847b-4d78-999a-ecf7c0620c46)

准确的说是无法自动加载网盘里的字幕，使用的是阿里云盘第三方客户端，适配了mpv
![image](https://github.com/hooke007/MPV_lazy/assets/43110396/8162e03e-6b75-4106-9734-aea3f8203222)
老版本的mpv可以正常加载，新版不行

**Comment by hooke007** [comment]
我不使用这个东西。
